Added description for how to handle ACS/SharePoint

wobba opened the pull request:
As far as I know, SharePoint is not affected nor is there a migration strategy, so I added guidance in the table for SharePoint.

azurecla commented:
Hi __@daltskin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
@kundanap - could you please review this public repo contribution?

tysonn commented:
@daltskin & @kundanap: Actually, I need to close this per https://github.com/Azure/azure-content/issues/6002. @daltskin - I will log in that issue for later review.

wobba committed:
Added description for how to handle ACS/SharePoint

As far as I know, SharePoint is not affected nor is there a migration strategy, so I added guidance in the table for SharePoint.
